modules/validations/suspicious_name.js

```diff
@@ -45,9 +45,17 @@ export function validationSuspiciousName() {
     return false;
   }
 
-  function isGenericName(name, tags) {
+  /** @param {string} name @param {string} presetName */
+  function nameMatchesPresetName(name, presetName) {
+    if (!presetName) return false;
+
+    return name.toLowerCase() === presetName.toLowerCase();
+  }
+
+  /** @param {string} name @param {string} presetName */
+  function isGenericName(name, tags, presetName) {
     name = name.toLowerCase();
-    return nameMatchesRawTag(name, tags) || isGenericMatchInNsi(tags);
+    return nameMatchesRawTag(name, tags) || nameMatchesPresetName(name, presetName) || isGenericMatchInNsi(tags);
```

Comment:
How does this avoid warning about NSI entries that add a `name` matching the brand name? Does the validator accept it because the NSI preset’s name has a compound “Brand – Preset” format?

Reply:
It works because the whole validator does nothing if a feature has `brand:wikidata` or `operator:wikidata`, even the  `not:name` check won't run...

https://github.com/openstreetmap/iD/blob/8bb1d39/modules/validations/suspicious_name.js#L149-L151

Reply:
> even the `not:name` check won't run

Interesting, that sounds like a bug, worth reporting separately.